packages/react-components/src/components/ControlButtonTooltip.tsx

```diff
@@ -18,6 +23,15 @@ const hostStyles: Partial<ITooltipHostStyles> = { root: { display: 'inline-block
  * @private
  */
 export const ControlButtonTooltip = (props: ITooltipHostProps): JSX.Element => {
+  const theme = useTheme();
+  const calloutStyle: Partial<ICalloutContentStyles> = { root: { padding: 0 }, calloutMain: { padding: '0.5rem' } };
+
+  // Place callout with no gap between it and the button.
+  const calloutProps = {
+    gapSpace: 0,
+    styles: calloutStyle,
+    backgroundColor: isDarkThemed(theme) ? theme.palette.neutralLighter : ''
```

Comment:
Does this background color need to be updated for every tooltip we have? Or just control bar buttons. (Do we mind having inconsistent tooltip backgrounds?)

Reply:
Hmmm good question, let me ask Alex P

Reply:
Note: (we may not care about this but) if a customer is using fluent components alongside our components/composites, the existing tooltips on their site will not be consistent with ours if we update them all

Reply:
Updated background color for all tooltips